test/Libraries/DynamoPythonTests/DynamoPythonTests.csproj

```diff
@@ -44,30 +44,31 @@
     </Reference>
     <Reference Include="IronPython, Version=2.7.9.0, Culture=neutral, PublicKeyToken=7f709c5b713576e1, processorArchitecture=MSIL">
       <HintPath>..\..\..\src\packages\IronPython.2.7.9\lib\net45\IronPython.dll</HintPath>
-      <Private>True</Private>
+      <Private>False</Private>
```

Comment:
I don't think these changes are great....
It means the tests will run - sure, but now we have multiple projects copies assemblies to the same output path - and it will be hard to figure out where they are coming from later.

Reply:
whoops, I'm thinking of this backwards right?!

Reply:
@mjkkirschner hmm I dont think these need to be true. I also have seen these properties are in consistent for some other test projects

Reply:
yeah, I had it backwards.